### DISCUSSION TITLE
Discussion on new logo style

### DISCUSSION THREAD

**Discussion by UG-SEP** [Kushal997-das/Project-Guidance#160] [Q&A] [answered]
Hey @Kushal997-das,
Well you are right the bg color is not looking attractive so I change a little bit in it 
Have a look
![Project Guidance (1)](https://user-images.githubusercontent.com/75884061/121284539-a5902a80-c8fa-11eb-91f2-ee5c20b2fbf6.gif)
`If this is not looking too attractive then please tell what color should I apply`

**Comment by Kushal997-das** [comment, marked as answer]
@SamarpanCoder2002 What do you think should we add this ??

**Reply by UG-SEP** [reply]
> @SamarpanCoder2002 What do you think should we add this ??

reply @SamarpanCoder2002

**Reply by SamarpanCoder2002** [reply]
@UG-SEP @Kushal997-das  I think the background color should be GitHub black color(User Black color value of GitHub background)....otherwise color confliction not so good

**Reply by UG-SEP** [reply]
ok

**Reply by UG-SEP** [reply]
# Before
![Project guidance](https://user-images.githubusercontent.com/75884061/121701261-57dc1380-caee-11eb-9af8-333c0828a8c9.gif)
# After
![Project Guidance (2)](https://user-images.githubusercontent.com/75884061/121700699-ca002880-caed-11eb-8594-88f4e306fc3d.gif)
I put the black color in the background now how it is looking you can compare it from the first open

**Reply by Kushal997-das** [reply]
@UG-SEP Is it possible to use dark background ?

**Reply by UG-SEP** [reply]
there are near about 12  colors some are dark some are light

**Reply by Kushal997-das** [reply]
Use dark then @UG-SEP

**Reply by UG-SEP** [reply]
> ![Project Guidance (2)](https://user-images.githubusercontent.com/75884061/121700699-ca002880-caed-11eb-8594-88f4e306fc3d.gif)
#  This one

**Reply by Kushal997-das** [reply]
This is not dark okey for now you can use this :)
